### PR TITLE
Add source.address to EVENT ID 3

### PR DESCRIPTION
**Pull request by janniten**
## What does this PR do?
Add the source.adress field to EVENT ID 3
https://www.elastic.co/guide/en/ecs/current/ecs-source.html#field-source-address

## Why is it important?
During the revision of some windows detection rules I've found that Sysmon's event ID 3 does not populate the source.address field, so those rules won't work.
The affected rules are

- Remote Scheduled Task Creation
- Incoming Execution via PowerShell Remoting
- Remotely Started Services via RPC
- Potential SharpRDP Behavior
- WMI Incoming Lateral Movement
- Incoming Execution via WinRM Remote Shell

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js [hunk 1]
```diff
@@ -811,6 +811,11 @@ var sysmon = (function () {
                     from: "winlog.event_data.SourceIp",
                     to: "source.ip",
                     type: "ip",
+                },
+				{
```
Comment (efd6) [inline x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js:815]: Can you fix the indentation here please; tabs to 4×space.

**Comment by efd6** [comment]
Note that processing in winlogbeat processing is being moved to ingest in #29435, so it is probably worth holding off on these until that is merged.

File: x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js (hunk 1 above)
Comment (janniten) [reply x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js]: @efd6, Fixed!

**Comment by janniten** [comment]
> Note that processing in winlogbeat processing is being moved to ingest in #29435, so it is probably worth holding off on these until that is merged.

Hi @efd6, These are very simple changes that complements the  DR, if it could be merged soon It will be great. 
I was preparing a more complex change to security module (related to this https://github.com/elastic/ecs/pull/1420) that affects the related.user field (and helps to detect user addition to a group when it is donde locally) and for this one I'll wait till the .js be moved to an ingestion pipeline.
Thank you!

**Comment by efd6** [comment]
@janniten, the change to ingest processing has been merged so this and your other two changes will need to be migrated to that.

**Comment by janniten** [comment]
> @janniten, the change to ingest processing has been merged so this and your other two changes will need to be migrated to that.

Hi @efd6, no problem.
I'll close this two PRs and I'll put this changes into the IP. Is this correct?
Thank you

**Comment by efd6** [comment]
It should just be a `set` using `copy_from` [along these lines](https://github.com/elastic/beats/blob/827e152b350d63607c84221b1184244bb05c4672/x-pack/winlogbeat/module/sysmon/ingest/sysmon.yml#L346-L351).

**Comment by janniten** [comment]
> It should just be a `set` using `copy_from` [along these lines](https://github.com/elastic/beats/blob/827e152b350d63607c84221b1184244bb05c4672/x-pack/winlogbeat/module/sysmon/ingest/sysmon.yml#L346-L351).

Hi @efd6 
I've create only one of the original PRs, (for event ID 3) since the user id is being populating now for all events in the ingetsion pipeline [here](https://github.com/elastic/beats/blob/827e152b350d63607c84221b1184244bb05c4672/x-pack/winlogbeat/module/sysmon/ingest/sysmon.yml#L856-L860)

Do you know where can I download the latest build, so I can test the pipeline in my dev enviroment?
Thank you!

**Comment by efd6** [comment]
Using the integration test pipeline should pull a docker image with the latest build snapshot.

**Pull request closed without merging**